toduck/Source/Protocol/TDPopupPresentation.swift

```diff
@@ -0,0 +1,14 @@
+import UIKit
+
+protocol TDPopupPresent {
```

Comment:
p3

```suggestion
protocol TDPopupPresentable {
```
프로토콜 네이밍은 형용사가 좋습니다

Reply:
오 그렇군요. 좋은 지적감사합니다.
여기만 바꾸면 에러가 생기니 밑 extension까지 바꾸셔야 됩니다.